crates/cxx-qt-gen/src/generator/naming/qobject.rs

```diff
@@ -28,6 +31,7 @@ impl From<&ParsedQObject> for QObjectName {
 impl From<&Ident> for QObjectName {
     fn from(ident: &Ident) -> Self {
         Self {
+            ident: ident.clone(),
```

Comment:
Maybe `rust_struct.rust` but that is an Ident that in the future might change to something else. I thought it was better to hold the input rather than assuming one of the CombinedIdents doesn't change.